diagnostics/physics/mld_eval.py

```diff
@@ -74,39 +94,55 @@ def plot_mld_eval(pp_root):
         cmap, norm = get_map_norm('cet_CET_L20', levels)
         bias_cmap, bias_norm = get_map_norm('cet_CET_D9', bias_levels)
 
-    p = grid[0].pcolormesh(model_grid.geolon_c, model_grid.geolat_c, mom_mld_winter, cmap=cmap, norm=norm)
+    # Set projection of input data files so that data is correctly tranformed when plotting
+    # For now, sst_eval.py will only support a projection for the arctic and a projection for all other domains
+    if config['projection_data'] == 'NorthPolarStereo':
+        proj = ccrs.NorthPolarStereo()
+    else:
+        proj = ccrs.PlateCarree()
+
+    # Model
+    p = grid[0].pcolormesh(model_grid.geolon_c, model_grid.geolat_c, mom_mld_winter, cmap=cmap, norm=norm, transform=proj)
     grid[0].set_title('(a) Model')
     cbar = autoextend_colorbar(grid.cbar_axes[0], p)
     cbar.ax.set_xlabel('Mean Jan-Mar MLD (m)')
     cbar.ax.set_xticks(np.arange(0, 301, 100, dtype=int))
+    logger.info("Successfully plotted model data")
 
-    p = grid[1].pcolormesh(argo_grid['lon_b'], argo_grid['lat_b'], deboyer_mld_winter, cmap=cmap, norm=norm)
+    # DeBoyer
+    p = grid[1].pcolormesh(argo_grid['lon_b'], argo_grid['lat_b'], deboyer_mld_winter, cmap=cmap, norm=norm, transform=proj)
     grid[1].set_title('(b) dBM 2022')
     cbar = autoextend_colorbar(grid.cbar_axes[1], p)
     cbar.ax.set_xlabel('Mean Jan-Mar MLD (m)')
     cbar.ax.set_xticks(np.arange(0, 301, 100, dtype=int))
+    logger.info("Successfully plotted DeBoyer data")
 
-    p = grid[2].pcolormesh(argo_grid['lon_b'], argo_grid['lat_b'], delta_deboyer, cmap=bias_cmap, norm=bias_norm)
+    # Model - DeBoyer
+    p = grid[2].pcolormesh(argo_grid['lon_b'], argo_grid['lat_b'], delta_deboyer, cmap=bias_cmap, norm=bias_norm, transform=proj)
     grid[2].set_title('(c) Model - dBM')
     cbar = autoextend_colorbar(grid.cbar_axes[2], p)
     cbar.ax.set_xlabel('Difference (m)')
-    annotate_skill(mom_rg, deboyer_mld_winter, grid[2], dim=['lat', 'lon'], fontsize=8) # TODO: need to cosine weight
+    annotate_skill(mom_rg, deboyer_mld_winter, grid[2], dim=['lat', 'lon'], fontsize=8, x0=config['text_x'], y0=config['text_y'], xint=config['text_xint'], plot_lat=config['plot_lat']) # TODO: need to cosine weight
```

Comment:
@andrew-c-ross, do you mean we can do something like this to account for cosine weight?
```
weight = np.cos(np.deg2rad(deboyer_mld_winter.lat))
weight = weight.broadcast_like(deboyer_mld_winter)
```